### PR TITLE
[RELEASE-1.11] Add performance tests target 

### PR DESCRIPTION
**Pull request by skonto**
**What this PR does / why we need it**:
- similar to https://github.com/openshift-knative/serving/pull/516

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,6 +30,14 @@ test-e2e:
 	./openshift/e2e-tests.sh
 .PHONY: test-e2e
 
+perf-tests:
+	export ES_DEVELOPMENT=true && \
```
Comment (ReToCode) [inline Makefile:34]: nit, spacing is a bit off here. Not sure if this is a problem or not.

File: Makefile [hunk 2]
```diff
@@ -30,6 +30,14 @@ test-e2e:
...
+    export ES_HOST_PORT=search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com && \
+    export USE_OPEN_SEARCH=true && \
```
Comment (ReToCode) [inline Makefile:36]: Do we need these to be exported? Probably
`ENABLE_INTERNAL_TLS="true" ./openshift/e2e-tests.sh`
like below would also work?

File: Makefile (hunk 1 above)
Comment (skonto) [reply Makefile:34]: This is weird because in my IDE it shows ok. I will check it.

File: Makefile (hunk 2 above)
Comment (skonto) [reply Makefile:36]: I will check.

**Comment by skonto** [comment]
Fixed the spaces issue, as discussed can't remove export due to the env lost with subsequent scripts called.

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve

**Pull request merged**